tests/rpmbuild.at

```diff
@@ -1656,6 +1656,25 @@ done
 [0],
 [expout],
 [])
+
+RPMTEST_CHECK([[
+cat << EOF > "${RPMTEST}"/tmp/bar.req
+#!/bin/sh
+fname=\$(head -n1)
+echo ";foo\$fname"
+echo "bar(\$(echo \$fname | xargs basename))"
+EOF
+
+touch $RPMTEST/file$i
+runroot ${RPM_CONFIGDIR_PATH}/rpmdeps --requires /file{1..5}
+echo "bar(file1)" >> expout
```

Comment:
This and following lines are unnecessary (I understand they're artifacts of the copy & paste of the previous test).